### ISSUE TITLE
[BUG] Direct Download link for windows client 0.9.9.8 goes to tar.gz not .zip

### ISSUE THREAD

**Issue by scycer** [valheimPlus/ValheimPlus#689]
See title

**Comment by prototype464** [comment]
You can still open it as a normal .zip file using WinRAR

**Comment by scycer** [comment]
Yeah u can, 7zip also kinda works but you gotta open the gz then inside open the tar. It's just not a common filettpe for windows users and I raised this as I had to troubleshoot this issue for a friend when they followed the instructions and instead of copying the inner contents to their windows client, they copied the tar file.

If this were server files, someone is likely more tech savvy and know, as it's the windows client then this is worth fixing (and also means it aligns with instructions)

**Comment by MartenBE** [comment]
My less tech savvy friends also had great difficulty with `.tar.gz` flies, it also doesn't correspond with some tutorials on youtube etc. Brings unneeded confusion unfortunately ...
